src/site/content/en/secure/what-is-mixed-content/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: What is mixed content?
+authors:
+  - johyphenel
+  - rachelandrew
+date: 2019-09-07
+updated: 2020-09-16
+description: >
+  Mixed content occurs when initial HTML is loaded over a secure HTTPS connection,
+  but other resources are loaded over an insecure HTTP connection.
+tags:
+  - security
+---
+
+**Mixed content** occurs when initial HTML is loaded over a secure HTTPS connection,
+but other resources (such as images, videos, stylesheets, scripts)
+are loaded over an insecure HTTP connection.
+This is called mixed content
+because both HTTP and HTTPS content are being loaded to display the same page,
+and the initial request was secure over HTTPS.
+
+Requesting subresources using the insecure HTTP protocol weakens the security of the entire page,
+as these requests are vulnerable to **man-in-the-middle attacks**,
+where an attacker eavesdrops on a network connection and views or modifies the communication between two parties.
+Using these resources, an attacker can often take complete control over the page,
+not just the compromised resource.
+
+Although many browsers report mixed content warnings to the user,
+by the time this happens, it is too late:
+the insecure requests have already been performed and the security of the page is compromised.
+
+This is why mixed content is increasingly being blocked by browsers.
+If you have mixed content on your site,
+then fixing it will ensure the content continues to load as browsers become more strict.
+
+## The two types of mixed content
+
+The two types of mixed content are: active and passive.
+
+**Passive mixed content** refers to content that doesn't interact with the rest of the page,
+and thus a man-in-the-middle attack is restricted to what they can do if they intercept or change that content.
+Passive mixed content includes images, video, and audio content,
+along with other resources that cannot interact with the rest of the page.
```

Comment:
The [spec](https://www.w3.org/TR/mixed-content/#category-optionally-blockable) defines passive mixed content as just images, video, and audio, so the last part of this sentence can be ommitted.